docs/Configuring-Pantheon/Networking.md

```diff
@@ -5,7 +5,21 @@ description: Pantheon networking is about P2P discovery and communication betwee
 
 Pantheon uses the network to find and connect to peers. 
 
-## Firewalls and Incoming Connections
+## Port Configuration 
+
+Ports must be exposed appropriately: 
+
+* Peer discovery port must be open on both TCP and UDP. The [`--p2p-port`](../Reference/Pantheon-CLI-Syntax.md#p2p-port) option specifies the
```

Comment:
Maybe something like:
To enable peer discovery, the p2p UDP port (--p2p-port) must be open for inbound connections.  

It is also highly recommended to open the p2p TCP port (--p2p-port) for inbound connections.  This is not strictly required as the pantheon node will try to initiate outbound TCP connections.  However, if no nodes on the network are accepting inbound TCP connections, nodes will not be able to communicate.

Reply:
Updated